Documentation/gettingstarted/aws-eni.rst

```diff
@@ -68,6 +68,62 @@ Deploy Cilium release via Helm:
 .. include:: namespace-kube-system.rst
 .. include:: hubble-enable.rst
 
+ENI Subnet tags
+===============
+
+To allow the Cilium Operator to filter subnets by tag in ENI mode, there are
+two pieces of configuration that must be provided, a custom CNI configuration
+and a field in the Cilium ``ConfigMap``:
+
+Create a CNI configuration
+--------------------------
+
+Create a `cni-config.yaml` file based on the template below. Fill in the
+``subnet-tags`` field:
+
+.. code:: yaml
```

Comment:
I think these all need to be `code-block`, `code` is rST while `code-block` is sphinx syntax which allows the sphinx-specific syntax highlighting to work correctly.

```suggestion
.. code-block:: yaml
```